Magical-Kitties/Magical-Kitties-Save-the-Day.html

```diff
@@ -1,173 +1,126 @@
+  <div class="masthead">
+    <span class="magical-kitties">Magical Kitties</span>
+    <span class="save-the-day">Save the Day!</span>
+  </div>
+  <h2 class="whiteonredtext" data-i18n="your-kitty-u"></h2>
+  <div class="redtext width200" data-i18n="kitty-name-u"></div>
+  <input type="text" class="grid-grow-2 input-large" name="attr_character_name">
 
-  <img src="https://raw.githubusercontent.com/Roll20/roll20-character-sheets/master/Magical-Kitties/magical-kitties-logo.png" class="logo" />
-  <br>
+  <!-- Trackables, Max & desc -->
+  <div class="trackables">
+    <h2 class="whiteonredtext trackables-list" data-i18n="trackables-u"></h2>
+    <h2 class="whiteonredtext trackables-limits" data-i18n="maximums-u"></h2>
+
+    <div class="redtext" data-i18n="owies-u"></div>
+    <input type="number" class="input-large" value="0" min="0" max="6" name="attr_owies">
+    <input type="number" class="input-large limit" placeholder="0" min="0" max="5" name="attr_owies_max">
+    <div class="redtext" data-i18n="treats-u"></div>
+    <input type="number" class="input-large" value="0" min="0" max="5" name="attr_treats">
+    <input type="number" class="input-large limit" placeholder="0" min="0" max="5" name="attr_treats_max">
+
+    <div class="redtext" data-i18n="injuries-u"></div>
+    <input type="number" class="input-large" value="0" min="0" max="4" name="attr_injuries">
+    <div class="nodisplay"></div>
+    <div class="redtext" data-i18n="level-u"></div>
+    <input type="number" class="input-large" value="0" min="0" max="10" name="attr_level">
+    <div class="nodisplay"></div>
+    <div class="redtext" data-i18n="current-xp-u"></div>
+    <input type="number" class="input-large" value="0" min="0" max="100" name="attr_xp">
+    <div class="nodisplay"></div>
+  </div>
 
- <div class="threequarterswide flex flex-center">
-    <h2 class="whiteonredtext" data-i18n="your-kitty-u"></h2>
+  <div class="halfwide">
+    <h2 class="whiteonredtext" data-i18n="kitty-description-u"></h2>
+    <textarea  name="attr_k_desc" class="kitty-desc" spellcheck="true"></textarea>
+  </div>
 
-    <div class="flex flex-between">
-      <span class="redtext width200" data-i18n="kitty-name-u"></span> <input type="text" class="flex-grow-2 input-large" name="attr_character_name">
-    </div>
 
-    <div class="flex flex-between">
-      <span class="redtext width200" data-i18n="extra-dice-rolls-u"></span> 
-      <button type="roll" class="button-roll" value="&{template:default} {{name=**Extra Dice**}} {{ Rolled ?{Extra Dice|1} extra dice.}} {{Successes = [[(?{Extra Dice})d6>?{Task Difficulty|Typical,4|Easy,3|Hard,5|Extreme,6}]] }} " data-i18n="roll-button-label"></button>
+  <!-- Attributes -->
+  <h2 class="whiteonredtext" data-i18n="attributes-u"></h2>
+  <div class="kitty-attributes">
+    <div class="score">
+      <label for="attr_cute_score" data-i18n="cute-u"></label>
+      <input type="number" placeholder="1" min="1" max="4" name="attr_cute_score">
```

Comment:
added `max="4"` since the maximum after level 4 is 4 in both editions, and `min="1"` since 1 is the minimum attribute value